.github/workflows/qa.yaml

```diff
@@ -80,6 +80,15 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version-file: go.mod
+      - name: Install VHS and ttyd for integration tests
+        run: |
+          set -eu
+          go install github.com/charmbracelet/vhs@latest
+
+          wget https://github.com/tsl0922/ttyd/releases/download/1.7.4/ttyd.x86_64
+          chmod +x ttyd.x86_64
+          sudo mv ttyd.x86_64 /usr/bin/ttyd
```

Comment:
The fact we require this being installed around makes things a bit complex when it comes to do local testing, it would be somewhat possible to automate this at test phase?

Reply:
Sure thing, not in the system... But we can just have a create path that we add to `PATH` for the duration of the test where the binary can be added if not available.

Or we should gracefully skip these tests if requirement components are not available.

Reply:
Tbh, this has become a development requirement for `authd` just as the Go and Rust toolchains, `protobuf` compilers, `docker` for `adsys`, and so on. I think it's kind of natural that tests fail due to the lack of required tooling and that any developer should install them to run the tests and make them pass. 

If we skip the tests due to the lack of a required component, changes can be done and uploaded without going through all our required tests (yes, I know that CI exists for this purpose, but alas). 

PS.: Depending on the `ttyd` version that will be available on Noble, we might need to skip these tests entirely (but it will be only for autopkgtests and launchpad builds)

Reply:
I see this is a required component, but the fact that it's not something that can be easily installed using normal go methods makes things harder to manage... I ideally would like that every component needed can be installed in a dev box without having to touch the system, but only affecting the dev environment.

However, speaking of autopkgtests, I think it's actually an example where it would be very important to have these tests running, probably also providing installed-tests, so that we ensure that when the system changes, everything still works as expected.